device-types/Supermicro/SYS-1019P-WTR.yaml

```diff
@@ -0,0 +1,71 @@
+---
+manufacturer: Supermicro
+model: SYS-1019P-WTR
+slug: sys-1019-wtr
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+comments: 'Specification: [https://www.supermicro.com/en/products/system/1U/1019/SYS-1019P-WTR.cfm](https://www.supermicro.com/en/products/system/1U/1019/SYS-1019P-WTR.cfm)'
+console-ports:
+  - name: COM1
+    type: de-9
+    label: Rear
+
+  - name: COM2
+    type: de-9
+    description: Header
+  - name: Front VGA
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+    description: USB 3.0
+  - name: Rear USB2
+    type: usb-a
+    description: USB 3.0
+  - name: Header USB3
+    type: usb-a
+    description: USB 3.0
+  - name: Header USB4
+    type: usb-a
+    description: USB 3.0
+  - name: Type A USB5
+    type: usb-a
+    description: USB 3.0
+  - name: Rear USB6
+    type: usb-a
+    description: USB 2.0
+  - name: Rear USB8
+    type: usb-a
+    description: USB 2.0
+  - name: Header USB9
+    type: usb-a
+    description: USB 2.0
+  - name: Header USB10
+    type: usb-a
+    description: USB 2.0
+  - name: Header USB11
+    type: usb-a
+    description: USB 2.0
+  - name: Header USB12
+    type: usb-a
+    description: USB 2.0
+  - name: Header USB13
+    type: usb-a
+    description: USB 2.0
```

Comment:
Invalid use of console server ports

Reply:
Corrected in https://github.com/netbox-community/devicetype-library/commit/44d267b3c9d05a6b202660c02221d13d5549e9a0

Reply:
Corrected in https://github.com/netbox-community/devicetype-library/commit/44d267b3c9d05a6b202660c02221d13d5549e9a0